src/libsystemd-network/sd-dhcp-client.c

```diff
@@ -116,6 +116,32 @@ static const uint8_t default_req_opts[] = {
         SD_DHCP_OPTION_DOMAIN_NAME_SERVER,
 };
 
+/* RFC7844 section 3:
+   MAY contain the Parameter Request List option.
+   RFC7844 section 3.6:
+   The client intending to protect its privacy SHOULD only request a
+   minimal number of options in the PRL and SHOULD also randomly shuffle
+   the ordering of option codes in the PRL.  If this random ordering
+   cannot be implemented, the client MAY order the option codes in the
+   PRL by option code number (lowest to highest).
+*/
+/* NOTE: using PRL options that Windows 10 RFC7844 implementation uses */
+static const uint8_t default_req_opts_anonymize[] = {
+       SD_DHCP_OPTION_SUBNET_MASK,                     // 1
+       SD_DHCP_OPTION_ROUTER,                          // 3
+       SD_DHCP_OPTION_DOMAIN_NAME_SERVER,              // 6
+       SD_DHCP_OPTION_DOMAIN_NAME,                     // 15
+       SD_DHCP_OPTION_ROUTER_DISCOVER,                 // 31
+       SD_DHCP_OPTION_STATIC_ROUTE,                    // 33
+       SD_DHCP_OPTION_VENDOR_SPECIFIC,                 // 43
+       SD_DHCP_OPTION_NETBIOS_NAMESERVER,              // 44
+       SD_DHCP_OPTION_NETBIOS_NODETYPE,                // 46
+       SD_DHCP_OPTION_NETBIOS_SCOPE,                   // 47
+       SD_DHCP_OPTION_CLASSLESS_STATIC_ROUTE,          // 121
+       SD_DHCP_OPTION_PRIVATE_CLASSLESS_STATIC_ROUTE,  // 249
+       SD_DHCP_OPTION_PRIVATE_PROXY_AUTODISCOVERY,     // 252
```

Comment:
nitpick: we try to stick to C89 comments, i.e. /* */, not C++ comments such as //... but it doesn't really matter much...